src/pyodide/internal/workers.py

```diff
@@ -895,7 +895,7 @@ def wrapper(*args, **kwargs):
 
 
 def _wrap_attr(attr):
-    if not callable(attr):
+    if not callable(attr) or isinstance(attr, (classmethod, staticmethod)):
```

Comment:
Of course, we learned that `isinstance(attr, classmethod)` implies `not callable(attr)` so checking that is redundant. But doing so is more readable. Maybe add a comment?